### DISCUSSION TITLE
Git commit is not available in R Studio.

### DISCUSSION THREAD

**Discussion by jjl1223** [community/community#131768] [Programming Help]
![에러1](https://github.com/community/community/assets/103105656/ce44f3e5-03e4-4504-a35d-3b40512dcdcf)
The following error occurs during the Rstudio commit process.

So I looked for the solution and added the address as follows.
![cmd](https://github.com/community/community/assets/103105656/7ef63d4f-614f-4887-85b7-ca4240c29f37)

However, the same error kept happening. So I restarted my computer, restarted Rstudio, and reinstalled git. Still, it didn't work out.

How can I fix the commit error?

**Comment by paiineli** [comment]
Hi,

It sounds frustrating to encounter persistent commit errors in RStudio. Here are some steps you can try to resolve the issue:

Check Git Configuration: Ensure that your Git configuration is set correctly. You can check your global config by running the following commands in the terminal:

git config --global user.name "Your Name"
git config --global user.email "your.email@example.com"

Update RStudio: Make sure you’re using the latest version of RStudio. Sometimes, updates can resolve compatibility issues.

Check Repository Path: Verify that you are in the correct directory of your Git repository when trying to commit. Use the terminal to navigate to your project folder.

Check Permissions: Ensure that you have the necessary permissions to write to the repository. If you're using a shared repository, check with the repository owner.

Use Terminal for Git Commands: Try committing your changes using the terminal instead of RStudio to see if the error persists:

git add .
git commit -m "Your commit message"


Review Error Messages: Pay close attention to any specific error messages that appear during the commit process. They can provide clues on what might be going wrong.

Reinstall RStudio and Git: If all else fails, consider completely uninstalling and then reinstalling both RStudio and Git, ensuring you remove all configuration files.

If the issue persists, please provide the specific error message you're encountering for more targeted assistance.

Best of luck!

**Reply by queenofcorgis** [reply]
@paiineli 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @jjl1223 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.


We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
